src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -254,6 +254,16 @@ public static bool IsWindowsAzureManagedHosting(this IEnvironment environment)
             return environment.IsAppService() && RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the application is running in Centauri environment.
```

Comment:
Let's avoid using Centauri project name, and use "Managed App Environment" instead